package/batocera/ports/devilutionx/devilutionx.mk

```diff
@@ -15,22 +15,11 @@ DEVILUTIONX_CONF_OPTS += -DCMAKE_BUILD_TYPE=Release
 DEVILUTIONX_CONF_OPTS += -DBUILD_TESTING=OFF
 # Prefill the player name when creating a new character,
 # in case the device does not have a keyboard.
-DEVILUTIONX_CONF_OPTS += -DPREFILL_PLAYER_NAME=ON                         
+DEVILUTIONX_CONF_OPTS += -DPREFILL_PLAYER_NAME=ON
 # Ensure that DevilutionX's vendored dependencies are not accidentally fetched from network.
 # They should all be present in the source package.
 DEVILUTIONX_CONF_OPTS += -DFETCHCONTENT_FULLY_DISCONNECTED=ON
 
-# ZeroTier on aarch64 uses ARMv8 Cryptography Extensions.
-# These extension are optional and only certain Armv8-A CPUs support them.
-# Disables ZeroTier if the CPU compilation flags do not specify the "+crypto" extension.
-ifeq ($(BR2_aarch64)$(BR2_ARM_CPU_ARMV8A),yy)
-    ifneq ($(findstring +crypto,$(BR2_TARGET_OPTIMIZATION)),+crypto)
-        DEVILUTIONX_CONF_OPTS += -DDISABLE_ZERO_TIER=ON
-    endif
-else ifeq ($(BR2_arm),y)
-    DEVILUTIONX_CONF_OPTS += -DDISABLE_ZERO_TIER=ON
```

Comment:
Note: Might have to bring this back for 32-bit if it causes build errors on some 32-bit arm systems

Reply:
are you not in a position to test?

Reply:
I don't have any devices currently

Reply:
I'm able to download and test. As of right now no online access on the rg40xxv. Was there an update? I can re-download and test.